src/scipp/format/formatter.py

```diff
@@ -0,0 +1,48 @@
+def format_variable(data, spec):
+    """
+    String formats the Variable according to the provided specification.
+    Parameters
+    ----------
+    obj
+        A scalar or array-like scipp Variable object
+    spec
+        Format specification; only 'c' for Compact error-reporting supported at present
+
+    Returns
+    -------
+    The formatted string
+    """
+    from scipp import Unit
```

Comment:
Can be moved to module scope using `from ..cpp_classes import Unit`

Reply:
I found that I need to modify this to `from ..core.cpp_classes import Unit`. Is this an indication that my testing environment is setup wrong?

Reply:
No, it just means that I don't know what I'm talking about :) Your version is correct.